library/crates/Cargo.toml

```diff
@@ -21,6 +23,8 @@ enum_dispatch = "=0.3.8"
 futures = { version = "=0.3.21", features = ["executor", "thread-pool"] }
 itertools = "=0.10.3"
 log = "=0.4.8"
+mime = "0.3.16"
```

Comment:
Add MIME type library so that TypeDB Cloud can append the correct MIME metadata when serving static files.